configs/swarm/docker-compose.local.yml

```
@@ -13,16 +13,18 @@ services:
       P2P_ADDRESS: 0.0.0.0:1337
       API_ADDRESS: 0.0.0.0:8080
       GENESIS_PUBLIC_KEY: ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4
-      GENESIS_PRIVATE_KEY: 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4
-      GENESIS_FILE: /config/genesis.json
       SUMERAGI_TRUSTED_PEERS: '[{"address":"irohad1:1338","public_key":"ed012083C85E315776FD2DDC187ECB23E608F800B313A1D614B108078EC048D5013D2D"},{"address":"irohad2:1339","public_key":"ed0120A37B7B758C952FE9429E9E35D1D71E2D8BB9364EDD077B5027ABAAC798D3230E"},{"address":"irohad3:1340","public_key":"ed0120B23E14F659B91736AAB980B6ADDCE4B1DB8A138AB0267E049C082A744471714E"}]'
     ports:
     - 1337:1337
     - 8080:8080
     volumes:
     - ./:/config
     init: true
-    command: irohad --submit-genesis
+    command: |-
+      /bin/sh -c "
+      kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file /tmp/signed-genesis.scale &&
+      GENESIS_SIGNED_FILE=/tmp/signed-genesis.scale irohad --submit-genesis
+      "
```

Wouldn't it work just the same?

```suggestion
    command: |
      kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file /tmp/signed-genesis.scale
      GENESIS_SIGNED_FILE=/tmp/signed-genesis.scale irohad --submit-genesis
```

---

It will be threated as one command (`GENESIS_SIGNED_FILE=/tmp/signed-genesis.scale irohad --submit-genesis` will be threated as three additional arguments of `kagami`)